validphys2/src/validphys/tests/test_pseudodata.py

```diff
@@ -80,3 +81,22 @@ def test_read_matches_recreate():
         )
         pd.testing.assert_index_equal(read.tr_idx, recreate.tr_idx, check_order=False)
         pd.testing.assert_index_equal(read.val_idx, recreate.val_idx, check_order=False)
+
+
+def test_make_level0_data():
+    from validphys.loader import Loader
+    from validphys.covmats import dataset_t0_predictions
+
+    dataset='NMC'
+    pdfname='NNPDF40_nnlo_as_01180'
+    theoryid=162
```

Comment:
You can take that information from the `conftest.py` (if there's already a set that contains that, otherwise you can create it)